configs/config_Pa_sample_combat.tsv

```diff
@@ -0,0 +1,11 @@
+local_dir	"/home/alexandra/Documents/Data/Batch_effects/"
+dataset_name	"Pseudomonas_analysis"
+analysis_name	"Pa_sample_lvl_sim"
+NN_architecture	"NN_2500_30"
+num_simulated_samples	6000
+lst_num_experiments	[1, 2, 5, 10, 20, 50, 100, 500, 1000, 2000, 3000, 5000]
```

Comment:
Does this and configs/config_Pa_experiment_combat.tsv need to be separate files? They're identical except for two lines. Would it be possible to combine them into one file, maybe using different names for the num_simulated_samples=600 and num_simulated_samples=6000 terms?